src/constants/colors.js

```diff
@@ -32,6 +32,8 @@ export const headerBgColor = "#A5A5A5";
 export const transparent = "transparent";
 
 export const textColor = "#272727";
+export const inputFieldBorderColor = "rgb(204, 204, 204)";
+export const inputFieldPlaceholderColor = "rgb(204, 204, 204)";
```

Comment:
Can we avoid this abstraction and instead name as the color from Zeplin? If a named color does not exist, chat with design and get them to add it to Zeplin. Ideally when creating our components we should just be following the design, and using the same language. I.e. the inputField border is "lightGrey", so in our inputField component, we see `borderColor: lightGreyColor`.

There is a couple of nice side effects of doing this:
* We can do a find across the project and inform design as to all the components that use a given color, enabling them to make more informed decisions if these need to change in future.
* Design can actually check the code for our components and ensure the correct colors are used.

Reply:
This wasn't a named color in Zeplin, yet. I will check with design which one to use or if we create a new one.

Reply:
TY

Reply:
@JunBadger Will add a new medium gray color, which we can use for disabled buttons, the input field border and placeholder color.